src/fundus/publishers/de/vogue.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from typing import List, Optional
+from lxml.etree import XPath
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+class VogueParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath("//article//p")
+        _subheadline_selector = XPath("//article//h2")
```

Comment:
This newspaper seems to support a little `summary` as well. An example would be [this article](https://www.vogue.de/artikel/jennifer-garner-outfit-hellblau-rot-sommer-2024). In Fundus context the summary would be
```
Hellblau und Rot wirken als Trendfarben allein schon stilvoll – Jennifer Garner beweist nun, wie genial die Kombination der beiden im Frühjahr/Sommer 2024 sein kann.
```

Further the selectors  are a bit too unspecific, extracting a lot of text unrelated to the article like

```
Von Katharina Fuchs
Mehr Themen auf Vogue.de:
Von Katharina Fuchs
Von Katharina Fuchs
Von Gabi Thorne
Von Marie Périer
Von Héloïse Salessy, Vogue Germany
Von Alessandra Signorelli
```

All of those are taken from the article I mentioned above.